lib/turn.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/turn'
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+
+class Turn
+    attr_reader :cards, :turns
+
+  def initialize(cards, turns)
+    @cards = card
+    @turns = turn
+  end
+
+  def guess
+    @turns.guess #need this to return guess
+  end
+
+  def number_correct?
+    @cards == correct
+  end
+
+  def feedback
+    @cards.none? #unsure of what method to call here
+  end
```

Comment:
In this method, we want to check if the guess is equal to the cards answer. If it is, then this method should return appropriate feedback like "Correct!" and "Wrong!"